change/@itwin-eslint-plugin-e19a7842-1ecb-4de7-b06d-ed44c1e4108b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Update `no-internal` rule checks to address some blind-spots",
```

Comment:
what "blind spots"? be more explicit here

Reply:
Could you update the PR description with a quick summary of those blind spots?